JS/Lesson14/sinan/script.js

```diff
@@ -4,90 +4,119 @@
 // Math.random() generate random number between 0 and 1 (not including 1)
 // Math.floor() 
 
-const randomNumberFrom0To10 = Math.floor(Math.random()*10+1);
+const randomNumber= Math.floor(Math.random()*10);
+console.log('Exercises bonus:',randomNumber);
+
 
-console.log(randomNumberFrom0To10);
 // 1. Use the `length` property to find the length of a string and print it to the console.
-const stringExercise1 = 'Object-oriented programming is a powerful paradigm.';
-console.log("Exercise1: ", stringExercise1.length);
 
-console.log('Object-oriented programming is'.length);
+const stringExercise1 = "Build a Personal Portfolio Webpage";
+console.log('Exercises 1:' , stringExercise1.length);
+
+
 
 console.log('------------------------');
 // 2. Use the `slice` method to extract 'Debugging code' from the string below and print it to the console.
 const stringExercise2 = 'Debugging code can be a frustrating but necessary task.';
-console.log("exercise2: ", stringExercise2.slice(0,14));
+console.log('Exercises 2.1:',stringExercise2.slice(0,13));
+console.log('Exercises 2.2:',stringExercise2.slice(0,stringExercise2.length));
```

Comment:
You only need to print the first statement (`console.log('Exercises 2.1:',stringExercise2.slice(0,13));`)